src/main/java/com/codingnomads/betty/logic/services/SentimentAnalyserService.java

```diff
@@ -0,0 +1,93 @@
+package com.codingnomads.betty.logic.services;
+
+import com.codingnomads.betty.logic.models.SentimentClassification;
+import com.codingnomads.betty.logic.models.SentimentResult;
+import edu.stanford.nlp.ling.CoreAnnotations;
+import edu.stanford.nlp.neural.rnn.RNNCoreAnnotations;
+import edu.stanford.nlp.pipeline.Annotation;
+import edu.stanford.nlp.pipeline.StanfordCoreNLP;
+import edu.stanford.nlp.sentiment.SentimentCoreAnnotations;
+import edu.stanford.nlp.trees.Tree;
+import edu.stanford.nlp.util.CoreMap;
+import org.apache.commons.lang3.StringUtils;
+import org.ejml.simple.SimpleMatrix;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Properties;
+
+@Service
+public class SentimentAnalyserService {
+
+    @Autowired
+    @Qualifier("sentimentProperties")
+    private Properties properties;
+    @Autowired
+    private StanfordCoreNLP pipeline;
+
+    public List<SentimentResult> sentimentResultList = new ArrayList<>();
+
+    public SentimentResult getSentimentResult(String text) {
+        validateInput(text);
+
+        return getSentimentResult(pipeline.process(text));
+    }
+
+    private void validateInput(String text) {
+        if (StringUtils.isBlank(text)) {
+            throw new InvalidInputException("Invalid input length");
+        }
+    }
+
+    private SentimentResult getSentimentResult(Annotation annotation) {
+
+        SentimentResult sentimentResult = null;
+        for (CoreMap sentence : annotation.get(CoreAnnotations.SentencesAnnotation.class)) {
+            sentimentResult = new SentimentResult();
+
+            Tree tree = sentence.get(SentimentCoreAnnotations.SentimentAnnotatedTree.class);
+            sentimentResult.setSentimentType(sentence.get(SentimentCoreAnnotations.SentimentClass.class));
+            sentimentResult.setSentimentClass(getClassification(RNNCoreAnnotations.getPredictions(tree)));
+            sentimentResult.setSentimentScore(RNNCoreAnnotations.getPredictedClass(tree));
+            sentimentResultList.add(sentimentResult);
+
+            printSentimentResultBySentence(sentimentResult, sentence);
+        }
+        //TODO: Returning the last result instead of a summary?!?!
```

Comment:
Remove comment :)

Reply:
Do you still need this comment? I can delete it if you dont